.github/workflows/ciwheel.yml

```diff
@@ -22,11 +25,12 @@ jobs:
 
       - name: Build wheels Linux
         if: ${{ startsWith(matrix.os, 'ubuntu') }}
-        uses: pypa/cibuildwheel@v1.9.0
+        uses: pypa/cibuildwheel@v2.3.0
         with:
           output-dir: dist
         env:
-          CIBW_BEFORE_BUILD: python -m pip install numpy setuptools
+          CIBW_BEFORE_ALL: yum -y install libxml2-devel
+          CIBW_BEFORE_BUILD: python -m pip install numpy setuptools scikit-build ninja cmake
```

Comment:
This will install `numpy`, I thought we wanted to avoid that? Why don't we add these to `build-system.requires`  in `pyproject.toml`. Does `cibuildwheel` not respect those?

Reply:
We can (try to) remove them. At some point they needed to be there though, but I don't remember why.